gui/src/app/view/settings.rs

```diff
@@ -244,18 +249,59 @@ pub fn bitcoind_edit<'a>(
 
     col = col
         .push(
-            Column::new()
-                .push(text("Cookie file path:").bold().small())
+            [RpcAuthType::CookieFile, RpcAuthType::UserPass]
+                .iter()
+                .fold(
+                    Row::new()
+                        .push(text("RPC authentication:").small().bold())
+                        .spacing(10),
+                    |row, auth_type| {
+                        row.push(radio(
+                            format!("{}", auth_type),
+                            *auth_type,
+                            Some(*selected_auth_type),
+                            SettingsEditMessage::BitcoindRpcAuthTypeSelected,
+                        ))
+                        .spacing(30)
+                        .align_items(Alignment::Center)
+                    },
+                ),
+        )
+        .push(match selected_auth_type {
+            RpcAuthType::CookieFile => Column::new()
                 .push(
-                    form::Form::new_trimmed("Cookie file path", cookie_path, |value| {
-                        SettingsEditMessage::FieldEdited("cookie_file_path", value)
-                    })
+                    form::Form::new_trimmed(
+                        "Cookie file path",
+                        &rpc_auth_vals.cookie_path,
+                        |value| SettingsEditMessage::FieldEdited("cookie_file_path", value),
+                    )
                     .warning("Please enter a valid filesystem path")
                     .size(20)
                     .padding(5),
                 )
                 .spacing(5),
-        )
+            RpcAuthType::UserPass => Column::new()
+                .push(
+                    Row::new()
+                        .push(
+                            form::Form::new_trimmed("User", &rpc_auth_vals.user, |value| {
+                                SettingsEditMessage::FieldEdited("user", value)
+                            })
+                            .warning("Please enter a valid user")
+                            .size(20)
+                            .padding(5),
+                        )
+                        .push(
+                            form::Form::new_trimmed("Password", &rpc_auth_vals.password, |value| {
+                                SettingsEditMessage::FieldEdited("password", value)
+                            })
+                            .warning("Please enter a valid password")
+                            .size(20)
+                            .padding(5),
+                        ),
```

Comment:
nit: can we add a small marging in-between the two text inputs?

Reply:
I've now added `spacing(10)` between them in both settings and installer.